Ran into the
```
Caused by: org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "originalfile_repo_path_index"
  Detail: Key (repo, regexp_split_to_array((('/'::text || path) || name) || '/'::text, '/+'::text))=(a5a2b4b3-d146-4ccc-98fa-7353a1ba023b, {"",user-3_9,2018-08,30,13-50-28.993,""}) already exists.
```
problem whilst testing this. 

Also had a DatabaseBusyException which appeared to freeze the server (couldn't log in via CLI or web) for a few minutes
```
2018-08-30 12:50:20,917 138482     [-thread-51] ERROR        ome.formats.importer.ImportLibrary - Error on import: Cannot acquire connection
omero.DatabaseBusyException: null
```

--- a/components/blitz/src/ome/formats/importer/ImportLibrary.java
+++ b/components/blitz/src/ome/formats/importer/ImportLibrary.java
@@ -271,7 +271,7 @@ public class ImportLibrary implements IObservable
         if (containers != null) {
             final int count = containers.size();
             ExecutorService filesetThreadPool, uploadThreadPool;
-            filesetThreadPool = Executors.newFixedThreadPool(config.parallelFileset.get());
+            filesetThreadPool = Executors.newFixedThreadPool(Math.min(count, config.parallelFileset.get()));
             uploadThreadPool  = Executors.newFixedThreadPool(config.parallelUpload.get());
             try {
                 final List<Callable<Boolean>> threads = new ArrayList<>(count);

